examples/app-router/cacheHandler.mjs

```diff
@@ -1,5 +1,9 @@
-import { Cache, FileSystemCache } from '@dbbs/next-cache-handler-core'
+import { Cache } from '@dbbs/next-cache-handler-core'
 
+import { FileSystemCache } from '@dbbs/next-cache-handler-core'
```

Comment:
```suggestion
import { Cache, FileSystemCache } from '@dbbs/next-cache-handler-core'
```

no need to import twice

Reply:
I thought about adding some examples in there. But let's keep it simple.